KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/OldCode.java

```diff
@@ -11,6 +11,40 @@
  */
 public class OldCode {
     
+    /*
+    /**
+     * Search for cards of a certain type in the current player's inventory and
+     * return if they are present Uses a linear search to find the type of card,
+     * and how many copies must be found.
+     *
+     * @param type What resource type to look for
+     * @param count How many cards of the type must be found to return true
+     * @return If the user has the given number of the type of cards
+     /
+    private boolean findCards(int type, int count) {

```

Comment:
Storing this old card searching method here, just in case

Reply:
Yes! Send old code to the File ref class